core/model/auth.go

```diff
@@ -109,6 +109,18 @@ type LoginSession struct {
 
 	DateUpdated *time.Time
 	DateCreated time.Time
+
+	AccountSummary []AccountSummary
```

Comment:
One login session can be related with only one account(or 0 if anonymous), so you should use *AccountSummary